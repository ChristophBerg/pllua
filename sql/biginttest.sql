CREATE FUNCTION int64_minus_one(value bigint)
RETURNS bigint AS $$
  return value - 1;
$$ LANGUAGE pllua;
select int64_minus_one(9223372036854775807);
