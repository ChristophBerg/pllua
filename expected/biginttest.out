CREATE FUNCTION int64_minus_one(value bigint)
RETURNS bigint AS $$
  return value - 1;
$$ LANGUAGE pllua;
select int64_minus_one(9223372036854775807);
   int64_minus_one   
---------------------
 9223372036854775806
(1 row)

